index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link rel="stylesheet" href="styles.css">
+    <title>Build-a-Bear</title>
+  </head>
+  <body>
+    <header>
+      <h1>Title</h1>
+    </header>
+    <main>
+      <!-- column1 -->
+      <section class="column1">
+        <article class="hats-selection">
+          hat buttons
+        </article>
+        <article class="cloths-selection">
```

Comment:
Everything looks great! Noticed a little spelling error here, not sure if it was on purpose or not. Again, otherwise looks fantastic!

Reply:
Good catch.  It was not on purpose, I just misspelled the word.  It's good to catch these little details early so that we won't miss these spelling errors in the future.